.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2023-08-24 01:23:51 UTC using RuboCop version 1.55.1.
+# on 2023-11-20 14:54:51 UTC using RuboCop version 1.55.1.
```

Comment:
I re-ran `rubocop --auto-gen-config` to see if there were any previously ignored rules that are no longer relevant. There were none, but commit this anyways to ensure this remains relevant and current.